_posts/2020-02-23-JabRef5-0.md

```diff
@@ -0,0 +1,41 @@
+---
+title:  JabRef 5.0 Major Release 
+id: jabref-5-0
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)" 
+bg: jabref-font
+color: white
+---
+
+We are proud to announce the release the next milestone of JabRef. Today we are releasing Jabref 5.0.
+The major difference is the completely reworked UI technology.
+
+
+## Installation
+
+With the release of the 5.0-beta we switched to a new native Java build system which allows us to bundle a custom Java runtime together with JabRef.
+This means that problems with other installed or outdated java versions on the system are a problem of the past. This also means it's no longer possible to start JabRef directly from the jar.  
+For the installation we now offer self-containing `.deb` and `.rpm` files for Linux users and in addtion to the usual installer for Windows.
+We also offer a portable zip version of JabRef which also includes all necessary files to run JabRef.
+No installed Java version required.
+
+## Read Before Installing
+
+Since Jabref 5.0 is a Major Release, you should be aware that not all features are fully compatible with older version of JabRef
+
+* TODO: Sorting of columns!?
+* Prefs Backward compatibility?! https://github.com/koppor/jabref/issues/364
+
+
+## Special Thanks
+
+The Core [JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS) would like to give special thanks to some temporary contributors who have significantly contributed to this release. You rock!
```

Comment:
I would also mention that we got a lot of contributions from many new contributors.